Assets/Scripts/WaveManager.cs

```diff
@@ -96,12 +98,20 @@ public void ReportDeath(GameObject entity)
             GameObject scoreField = GameObject.FindGameObjectsWithTag("Score")[0];
             ScoreModel view = scoreField.GetComponent<ScoreModel>();
             view.Increment(20);
-            
+            generateDrop(entity.transform.position);
             Destroy(entity);
             minions.Remove(entity);
         }
     }
 
+    private void generateDrop(Vector3 position)
+    { 
+        if (UnityEngine.Random.Range(0, 4) == 1) // 25% drop chance (maybe 20%, too lazy to look up)

```

Comment:
Did the research, the upper limit is exclusive.
Hence, the chance is 25%, so please change the comment.
Furthermore, it would be nice and idiomatic to check if the result is zero instead of one.